components/__tests__/search-filter.test.tsx

```diff
@@ -32,58 +32,11 @@ describe("SearchFilter", () => {
       />,
     );
 
-    expect(container.firstChild).toMatchInlineSnapshot(`
-      <div
-        class="grid grid-cols-2 gap-4"
-      >
-        <div
-          class="space-y-1"
-        >
-          <label
-            class="block text-sm font-medium text-gray-700"
-            for="filter-kebutuhan"
-          >
-            Kategori
-          </label>
-          <select
-            class="inline-block shadow-sm focus:ring-blue-500 focus:border-blue-500 w-full text-sm border-gray-300 rounded-md"
-            id="filter-kebutuhan"
-            name="kebutuhan"
-            title="Kategori"
-          >
-            <option
-              value=""
-            >
-              Semua
-            </option>
-            <option
-              value="Ambulans"
-            >
-              Ambulans
-            </option>
-            <option
-              value="Donor plasma"
-            >
-              Donor plasma
-            </option>
-            <option
-              value="Kontak penting"
-            >
-              Kontak penting
-            </option>
-            <option
-              value="Rumah sakit"
-            >
-              Rumah sakit
-            </option>
-          </select>
-        </div>
-      </div>
-    `);
+    expect(screen.getByText(/kategori/gi)).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByText(/kategori/i)).toBeVisible();
```

Please use `.toBeVisible()` assertion instead to ensure that the element is actually visible.
And also avoid the `global` flag in the regex, as I have never gotten it recommended in any @testing-library examples, including the ones I'm getting  from testing-playground.com.